scrapy_zyte_api/handler.py

```diff
@@ -129,6 +131,7 @@ def _build_client(settings):
                 api_key=settings.get("ZYTE_API_KEY") or None,
                 api_url=settings.get("ZYTE_API_URL") or API_URL,
                 n_conn=settings.getint("CONCURRENT_REQUESTS"),
+                user_agent=f'{settings.get("_USER_AGENT", "")} {USER_AGENT} {PYTHON_ZYTE_API_USER_AGENT}'.lstrip(),
```

Comment:
Let's name the setting _ZYTE_API_USER_AGENT

Reply:
It also looks like _ZYTE_API_USER_AGENT should overwrite the user_agent completely, like in python-zyte-api.

Reply:
Alternatively, you can call the setting _ZYTE_API_USER_AGENT_EXTRA or something like that, though overwriting completely sounds a bit better to me

Reply:
What about calling it `ZYTE_API_USER_AGENT`, without the `_` prefix, and simply not document it?

Reply:
My concern is that users might discover it, and think it's a user-agent to send to the remote websites, similar to USER_AGENT setting.

Reply:
Underscore makes it look a bit more scrary at least :)

Reply:
Love “scrary” as a concept :slightly_smiling_face: